### PR TITLE
Fix config protos in trillian-examples Gossip

### PR DESCRIPTION
**Pull request by Martin2112**
Affects tests only.

### CODE REVIEW

**Comment by Martin2112** [comment]
Towards github.com/google/trillian#1738.

File: gossip/hub/instance_test.go [hunk 1]
```diff
@@ -56,7 +56,7 @@ func TestConfigFromSingleFile(t *testing.T) {
 	tests := []struct {
 		desc, contents string
 		missingFile    bool
-		want           configpb.HubMultiConfig
+		want           *configpb.HubMultiConfig
```
Comment (daviddrysdale) [inline gossip/hub/instance_test.go]: Why does this need to change?  I can't immediately see anywhere where `test.want` gets passed by value

Comment (Martin2112) [reply gossip/hub/instance_test.go]: The loop over the tests does it via the assignment to the loop variable. That copies the outer struct, which copies the embedded proto.

Comment (daviddrysdale) [reply gossip/hub/instance_test.go]: Ah, got it, thanks.

**Pull request merged**